api_callable.go

```diff
@@ -0,0 +1,98 @@
+package gax
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"time"
```

Comment:
I also suggest to run gofmt command. It can take care of those imports or minor style errors.